src/main/java/com/afwsamples/testdpc/common/Util.java

```diff
@@ -354,4 +355,112 @@ public static String constantToString(Class<?> clazz, String prefix, int value)
   private static String constNameWithoutPrefix(String prefix, Field field) {
     return field.getName().substring(prefix.length());
   }
+
+  public static String getStringEventTagFromId(int eventId) {
```

Comment:
I think you forgot to remove this? (it can also be statically referenced)

Reply:
Ok, I reverted all the changes to `Util.java` and made the two methods in `SecurityLogsFragment.java` static. I've also cleanup the formatting mess that my editor did in `ShellCommand.java`.
The diff looks a bit cleaner now.